cppwinrt/component_writers.h

```diff
@@ -175,6 +175,9 @@ void* __stdcall %_get_activation_factory([[maybe_unused]] std::wstring_view cons
 int32_t __stdcall WINRT_CanUnloadNow() noexcept
 {
 #ifdef _WRL_MODULE_H_
+#ifdef _MSC_VER
+#pragma warning(disable: 4324) // structure was padded due to alignment specifier
```

Comment:
If possible it is preferrable to pragma warning push/pop so it is only disabled for the small bit of code.  If there is one line of warning then `pragma warning (suppress: 4324)` disables it for only the next line.

Reply:
Good catch. This one was an oversight on my part. (Had to do a couple other push/disable/pop elsewhere because, compiler yuckiness, and got into a pattern, haha)